components/member-profile/contribution/dates/index.js

```diff
@@ -0,0 +1,65 @@
+import PropTypes from 'prop-types';
+import { timeWas } from '../../../../helper-functions/time-was';
+
+const Dates = (props) => {
+  const { isTaskAvailable, startedOn, endsOn, status, what, allContributionObj } = props;
+  let completedText = 'Completed in:';
+  let completedDate = '';
+  let featureLiveOnText = '';
+  let featureLiveDate = '';
+
+  if (isTaskAvailable) {
+    if (status === 'Active') {
```

Comment:
Aren't we having `Active` in lowercase ?
Thank you! 😄

Reply:
Not in this PR, we first need to change it in the backend. If I change it now, the frontend will break and will not have any contributions to show.
Will discuss this and get it changed in the backend.

Reply:
Okay! 👍